src/schema/v2/location.ts

```diff
@@ -106,6 +106,10 @@ export const LocationType = new GraphQLObjectType<any, ResolverContext>({
 
     display: {
       type: GraphQLString,
+      resolve: ({ address, city, state, country }) =>
+        `${address && address + ", "}${city && city + ", "}${
+          state && state + ", "
+        }${country && country}`,
```

Comment:
- For Gravity `Location` that already has its `display`, we can just use it and not break existing behavior.
- For formatting addresses from parts, we can do something similar to [`Location`](https://github.com/artsy/gravity/blob/b448fcf07b6d8871f0767d364fe792b2f5c407a4/app/models/domain/location.rb#L117).

So maybe something like 👇  (untested):

```typescript
resolve: ({ display, address, address_2, city, state, postal_code, country }) => {
  display ||
    compact([address, address_2, city, state, postal_code, country]).join(", ")
}
```

Also, some unit tests would be great!

Reply:
Thanks! I tried to use `compact` and it wasn't working, but now I see you have to explicitly return it 😅 

I'm not seeing a test file for `location.ts` do you know if this is tested in another file? Otherwise, I will create one.